{{cookiecutter.repostory_name}}/deploy.sh

```diff
@@ -2,10 +2,26 @@
 PRIMARY_IMAGE_HASH=$(docker-compose build app | tee /dev/tty \
     | sed '/^$/d' | grep "AS secondary-image" -B1 | head -1 | awk '{print $2}')
 docker-compose build  # build the rest of images
-docker-compose down -v --remove-orphans
-docker-compose up -d
+
+docker-compose stop app \
+{% if cookiecutter.use_celery == "y" -%}
+    celery-worker celery-beat \
+{%- endif %}
+{% if cookiecutter.use_flower == "y" -%}
+    celery-flower
+{%- endif %}
+
+docker-compose start app \
```

Comment:
https://stackoverflow.com/a/33716320/95920 says it won't create the container in the first place. Can we use `up`?

Reply:
Yes, actually I don't need to specify services when running `up`. Every single service needs to be up